master/internal/trials/utils.go

```diff
@@ -42,3 +42,32 @@ func CanGetTrialsExperimentAndCheckCanDoAction(ctx context.Context,
 	}
 	return nil
 }
+
+// CanGetTrialsExperimentAndCheckCanDoActionBulk functions the same as
+// CanGetTrialsExperimentAndCheckCanDoAction but takes in multiple trial ids.
+func CanGetTrialsExperimentAndCheckCanDoActionBulk(ctx context.Context,
```

Comment:
test?

Reply:
+1  to Sean's point.
Also `CanGetTrialsExperimentAndCheckCanDoAction` is over a year old at this point, and, if this is correctly categorized as a chore, then I think we should spend a little more time and improve the test coverage in this same ticket

Reply:
Added tests. Had to move the GetUser call outside of the function. Updated the functions that call `CanGetTrialsExperimentAndCheckCanDoAction ` as well.